crates/telio-wg/src/uapi.rs

```diff
@@ -468,6 +471,26 @@ impl Peer {
         }
         dual_ip_addresses
     }
+
+    /// Returns whether the peer is a virtual peer for multicast
+    pub fn is_multicast_peer(&self) -> bool {
+        for ips in &self.allowed_ips {
+            if ips.network().eq(&STARCAST_VPEER_INTERNAL_IPV4)
```

Comment:
Again, just check by internal ipv4.
Checking by multicast network does not really make sence